src/App/Pages/Accounts/LoginPageViewModel.cs

```diff
@@ -249,8 +253,7 @@ public async Task LogInAsync(bool showLoading = true, bool checkForExistingAccou
 
         private async Task MoreAsync()
         {
-            var emailExists = _stateService.AccountViews != null && _stateService.AccountViews.Any(e => e.Email == Email);
-            var buttons = IsEmailEnabled || !emailExists
+            var buttons = IsEmailEnabled || string.IsNullOrEmpty(await _stateService.GetUserIdAsync(Email))
```

Comment:
Can't you use `EmailIsInSavedAccounts` here? instead of making the `GetUserIdAsync` call

Reply:
Actually what i wanted was the **Active**UserEmail